tests/api/utils/entities.js

```diff
@@ -18,6 +22,21 @@ const entitiesUtils = module.exports = {
     .then(res => res.entities[uri])
   },
 
+  findOrIndexEntities: async (uris, index = 'wikidata') => {
+    const ids = _.map(uris, unprefixify)
+    const results = await Promise.all(ids.map(entitiesUtils.getElasticEntity(index)))
+    const entitiesFound = _.filter(results, _.property('found'))
+    const entitiesFoundUris = entitiesFound.map(_.property('_source.uri'))
+    const entitiesNotFoundUris = _.difference(uris, entitiesFoundUris)
+    if (_.isNonEmptyArray(entitiesNotFoundUris)) {
+      // index entities into elasticsearch by getting the uris
+      await entitiesUtils.getByUris(entitiesNotFoundUris)
+      await wait(elasticsearchUpdateDelay)
```

Comment:
```suggestion
      await Promise.all(ids.map(id => waitForIndexation('wikidata', id))
```
`waitForIndexation` (introduced in 5667a47a, so accessible after a rebase on `master`) is more reliable than an arbitrary `elasticsearchUpdateDelay`